src/Support/Coverage.php

```diff
@@ -42,6 +42,10 @@ public static function isAvailable(): bool
             return false;
         }
 
+        if ($runtime->hasPCOV()) {
+            return true;
```

Comment:
Think maybe we should also add a check for PHPDBG?

Reply:
Make sense, I'd improve readability also in that case but I'll try to change the minimum.